### PR TITLE
User controller/promise

### PR DESCRIPTION
**Pull request by KyuaKwon**
Update some function of user.controller.js using promise.

- signin
- storeSession
- crawlByAccessTokenFacebook

@ethkim @nlpenguin @ace0625

### CODE REVIEW

File: src/server/controllers/users.controller.js [hunk 1]
```diff
@@ -100,74 +101,88 @@ export function signin(req, res, next) {
...
-    (error, response, userBody) => {
-      if (!error && response.statusCode == 200) {  // if HTTP request&response successfully.
-        let result = JSON.parse(userBody);
-        // Crawl user profile_picture from facebook by access token.
-        request.get({
+      resolveWithFullResponse: true,
+    })
+      .then((facebookDataResult) => {
+        result = JSON.parse(facebookDataResult.body);
+        if (facebookDataResult.statusCode === 200 && result.verified === true) {  // if HTTP request&response successfully.
```
Comment (KyuaKwon) [inline src/server/controllers/users.controller.js]: maxLen == 100

**Comment by lyw0149** [comment]
SRDYHTSEESDRFTFDLGGFHLKJHLGTMGGEDTGVGDLTGFGSLTFDVGDFHGSFDVBAESBDTFX

**Comment by lyw0149** [comment]
:shipit:

File: src/server/controllers/users.controller.js [hunk 2]
```diff
@@ -131,43 +140,51 @@ function storeSession(req, user) {
...
+            resolveWithFullResponse: true,
           });
-      } else {
-        responseCallback();
-      }
-    });
+        }
+      })
+      .then((facebookPictureResult) => {
+        // if HTTP request&response successfully.
```
Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: Same comments in line 167, can you explain?

Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: Ok solved. LGTM

**Pull request merged**